test/sandbox.jl

```diff
@@ -1,11 +1,65 @@
+# <<<<<<< HEAD
+# Debug logging
+using Test
+using Logging
+using Neo4j
 using DistributedFactorGraphs
-struct TestInferenceVariable1 <: InferenceVariable end
-a = DFGVariable(:a, TestInferenceVariable1())
-b = copy(a)
+using IncrementalInference, RoME
+using Test, Dates
+# logger = SimpleLogger(stdout, Logging.Debug)
+# global_logger(logger)
 
-d1 = getSolverData(a)
-pop!(solverDataDict(a), :default)
-d2 = getSolverData(b)
+dfg = GraphsDFG{SolverParams}(;
+    userId="testUser", robotId="testRobot", sessionId="testSession",
+    params=SolverParams())
 
-d1.dims = 6
-d2.dims
+# Add the first pose :x0
+x0 = addVariable!(dfg, :x0, Pose2)
```

Comment:
Just an FYI, we now have `RoME.generateCanonicalFG_Hexagonal()` which should do the same job.